independent-publisher-connectors/SHRTCODE/apiDefinition.swagger.json

```diff
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SHRTCODE",
+    "description": "With this free Link Shortener you can make Links shorter and easier to remember.",
+    "version": "1.0",
+    "contact": {
+      "name": "Chandra Sekhar",
+      "url": "https://github.com/sekharmalla",
+      "email": "chandra.malla@msftcommunity.com"
+    }
+  },
+  "host": "api.shrtco.de",
+  "basePath": "/v2/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/shorten": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "succcess",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ok": {
+                  "type": "boolean",
+                  "description": "ok",
+                  "title": "OK",
+                  "enum": [true, false]
+                },
+                "result": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "type": "string",
+                      "description": "code",
+                      "title": "Code"
+                    },
+                    "short_link": {
+                      "type": "string",
+                      "description": "short_link",
+                      "title": "Short link"
+                    },
+                    "full_short_link": {
+                      "type": "string",
+                      "description": "full_short_link",
+                      "title": "Full short link"
+                    },
+                    "short_link2": {
+                      "type": "string",
+                      "description": "short_link2",
+                      "title": "Short link 2"
+                    },
+                    "full_short_link2": {
+                      "type": "string",
+                      "description": "full_short_link2",
+                      "title": "Full short link 2"
+                    },
+                    "share_link": {
+                      "type": "string",
+                      "description": "share_link",
+                      "title": "Share link"
+                    },
+                    "full_share_link": {
+                      "type": "string",
+                      "description": "full_share_link",
+                      "title": "Full share link"
+                    },
+                    "original_link": {
+                      "type": "string",
+                      "description": "original_link",
+                      "title": "Original link"
+                    }
+                  },
+                  "description": "result"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Shortening a Link",
+        "description": "Create a short link for a given URL. Requires a `url` parameter.",
+        "operationId": "ShorteningaLink",
```

Comment:
@sekharmalla operationId property should be in pascal case. You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).